certs/.gitignore

```diff
@@ -0,0 +1,2 @@
+*
+!.gitignore
```

Comment:
The line `!.gitignore` explicitly makes sure any changes to `.gitignore` are *not* ignored, but makes sure all other files in this subpath *are* ignored.  This is the standard way to make sure a path exists, while not exposing any contents of the path.